src/pages/auth/Login.js

```diff
@@ -1,19 +1,44 @@
 import React, { useState } from "react";
-import { Link,useNavigate} from "react-router-dom";
-import { AuthContainer } from "./component/AuthContainer";
+import { Link, useNavigate,useLocation } from "react-router-dom";
+import { AuthContainer } from "../auth/component/AuthContainer";
 import { SigninImage } from "../../assets/images";
 import {
   BsFillEyeFill,
   FaEnvelope,
   AiOutlineArrowRight,
+  BsFillEyeSlashFill,
 } from "../../assets/icons";
-import "./auth.css"
 
+import { useAuth } from "../../context/auth/authContext";
 
-const Login = () => {
-  const [email, setEmail] = useState();
 
-  const [password, setPassword] = useState();
+export const Login = () => {
+  const [showPassword, setShowPassword] = useState(false);
+  const {pathname} = useLocation();
+  
+  const navigate = useNavigate();
+  const { login } = useAuth();
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+    rememberMe : false
+  });
+  const handleOnChange = (e) => {
+    setFormData({ ...formData, [e.target.id]: e.target.value });
+  };
+
+  const loginHandler = async (e, email, password) => {
+    e.preventDefault();
+    const success = await login({ email, password });
+    if (success) {
+      if(pathname === "/signup"){
+        navigate("/");
+      }else{
+        navigate(-1);
+      }
+    }
```

Comment:
You can do this in the `login()` then  you don't need to check the success status here again.

Reply:
I will use this.